addons/adminMessages/functions/fnc_displayMessage.sqf

```diff
@@ -28,10 +28,19 @@ if (
 
 };
 
+
 // add message to chat either way >> user can check chat history on map, if enabled
-GVAR(channel) radioChannelAdd [player];
-GVAR(channel) radioChannelSetCallSign format [localize "STR_grad_ADMINMESSAGES_CHANNEL", _info];
+private _player = ace_player;   // ace_player so that a Remote Controlling Zeus see's the message.
+private _chID = GVAR(channel);
+
+_chID radioChannelAdd [_player];
+_chID radioChannelSetCallSign format [localize "STR_grad_ADMINMESSAGES_CHANNEL", _info];
```

Comment:
```suggestion
private _player = ace_player;   // ace_player so that a Remote Controlling Zeus see's the message.
private _channelID = GVAR(channel);

_channelID radioChannelAdd [_player];
_channelID radioChannelSetCallSign format [localize "STR_grad_ADMINMESSAGES_CHANNEL", _info];
```